### ISSUE TITLE
readTrees warning when reading from a MCC tree

### ISSUE THREAD

**Issue by abhitims** [cmt2/RevGadgets#32]
Hi I am getting this warning, when reading a single MCC tree. Any suggestions as to why this might be happening?

Warning message:
In .parseTreeString(tree_strings[i]) : No index found in tree file.
            This file may not work with downstream plotting functions.

thanks!

**Comment by cmt2** [comment]
The warning is just that, a warning (not an error). Feel free to proceed, if you experience issues downstream issues, please reach out again.

**Comment by abhitims** [comment]
I initially was facing some issues, which I thought was due to this but all good. Many thanks again, the package has been very useful.

One quick question, I wanted to just plot a subclade of interest from the entire MCC tree (say based on a MRCA node or a list of tip labels), I was wondering if there is any neat way to do it. I tried tinkering with the phylo and the data of the readTrees output but with limited success. Any suggestions on that will be much appreciated. Let me know if you would need an example and if I should put this up as a different issue as well.

Thanks  a bunch.
